changelog/issue-5694-new-deposits-card-skeleton

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: This PR is adding low level structural changes to implement deposits overview changes
```

Comment:
Does this comment go out in changelog, or does `patch` mean it's hidden? Could add the eventual changelog here, e.g. `New deposits overview card (behind feature flag)`

Reply:
No, this comment shouldn't get written to any changelogs.

When filling out this changelog, I chose an empty changelog message and it said that the changelog won't be written but it wanted to know a reason.